skorch/callbacks/training.py

```diff
@@ -532,7 +532,7 @@ def initialize(self):
         return self
 
     def named_parameters(self, net):
-        return net.module_.named_parameters()
+        return net.get_learnable_params()
```

Comment:
Unfortunately not, the names can be identical. I think this method is really just for the case that you don't need to discern. Otherwise, a user should use `self.module_.named_paramters()`, `self.mymodule_.named_parameters()` etc., as shown in the docs.

Reply:
Maybe we need to refactor this callback then, have it implement `named_parameters(self, net, module_name)` and retrieve the module from `net.modules_`. When specifying parameters we could then introduce `custommodule__0__foo=3` and have `custommodule` resolve the module object. But this does not need to happen right now. It is probably an uncommon combination anyway.

Reply:
Yes, good point. Shall I roll back the change here to preserve the current behavior?

Reply:
No I think this is fine as is, adding a `FIXME` might be a good idea so that we know that this is an issue and where it comes from.